src/js/components/awardv2/table/FederalAccountTable.jsx

```diff
@@ -0,0 +1,162 @@
+        const displayName = tableMapping.table[column];
+
+        const isLast = columnIndex === tableMapping.table._order.length - 1;
+
+        return (
+            <FederalAccountTableHeaderCell
+                column={column}
+                label={displayName}
+                order={this.props.sort}
+                defaultDirection={tableMapping.defaultSortDirection[column]}
+                setTransactionSort={this.props.changeSort}
+                isLastColumn={isLast} />
+        );
+    }
+
+    bodyCellRender(columnIndex, rowIndex) {
+        const column = tableMapping.table._order[columnIndex];
+        const item = this.props.fundingResults[rowIndex];
+
+        const isLast = columnIndex === tableMapping.table._order.length - 1;
+        let isLink;
+
+        if (column === 'id') {
+            isLink = `#/award/${item.id}`;
+        }
+        if (column === 'agency') {
+            isLink = `#/agency/${item.agencyId}`;
+        }
+        if (column === 'fedAccount') {
+            isLink = `#/federal_account/${item.agencyId}-${item.mainAccountCode}`;
+        }
+        return (
+            <FederalAccountTableGenericCell
+                rowIndex={rowIndex}
+                data={item[column]}
+                link={isLink}
+                isLastColumn={isLast} />
+        );
+    }
+
+    buildTable() {
+        let totalWidth = 0;
+
+        const columns = tableMapping.table._order.map((column, i) => {
+            const columnX = totalWidth;
+            const isLast = i === tableMapping.table._order.length - 1;
+
+            const displayName = tableMapping.table[column];
+            let columnWidth = Math.max(measureTableHeader(displayName),
+                tableMapping.columnWidths[column]);
+            if (isLast) {
+                // make it fill out the remainder of the width necessary
+                const remainingSpace = this.props.tableWidth - totalWidth;
+
+                columnWidth = Math.max(remainingSpace, columnWidth);
+            }
+
+            totalWidth += columnWidth;
+
+            return {
+                x: columnX,
+                width: columnWidth
+            };
+        });
+
+        return {
+            columns,
+            width: totalWidth
+        };
+    }
+
+    render() {
+        const tableValues = this.buildTable();
+
+        let loadingClass = '';
+        if (this.props.inFlight) {
+            loadingClass = 'loading';
+        }
+
+        let message = null;
+        if (this.props.fundingResults.length === 0) {
+            message = (<ResultsTableNoResults />);
+        }
+
+        return (
+            <div
+                className={`transactions-table ${loadingClass}`}
+                ref={(div) => {
+                    this.wrapperDiv = div;
+                }}>
+                <IBTable
+                    rowHeight={rowHeight}
+                    rowCount={this.props.fundingResults.length}
+                    headerHeight={50}
```

Comment:
In this case yes, because it's only used to calculate style properties in the custom table library (https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/components/sharedComponents/IBTable/components/Table.jsx). We could store a standard `headerHeight` that we use on most tables, but I think refactoring for that is outside the scope of this ticket.